testdata/parts/empty-ctype-bad-content.raw

```diff
@@ -1,7 +1,7 @@
 Content-Type: multipart/alternative; boundary="----=_Part_10541_215446765.98298273965074084"
 
 ------=_Part_10541_215446765.98298273965074084
-Content-Type: <html>
+Content-Type:
```

Comment:
This had to be adjusted as `<html>` was no longer failing validation due to `<` and `>` being removed in `fixMangledMediaType`